### DISCUSSION TITLE
Referência

### DISCUSSION THREAD

**Discussion by RogerioPiazzon** [maritaca-ai/maritalk-api#39] [General]
Olá ! 
Boa noite !
Algum artigo de referência que posso citar num trabalho acadêmico ? Vi somente o do sabia... Senão existe alguma citação padrão ?

**Comment by rodrigo-f-nogueira** [comment]
Olá @RogerioPiazzon, iremos publicar um technical report dentre 4-6 semanas que poderá ser usado como citação da MariTalk.

**Comment by hugoabonizio** [comment]
Olá @RogerioPiazzon! Você pode usar como referência o nosso [relatório técnico](https://arxiv.org/abs/2403.09887).

```bibtext
@article{maritaca2024sabia2,
  title={Sabi{\'a}-2: A New Generation of Portuguese Large Language Models},
  author={Sales Almeida, Thales and Abonizio, Hugo and Nogueira, Rodrigo and Pires, Ramon},
  year={2024}
}
```
